depthmapX/mainwindow.cpp

```diff
@@ -1573,8 +1573,8 @@ void MainWindow::MakeGraphTree()
             m_treeroots[0] = hItem;
         }
         QTreeWidgetItem* hItem = m_treeroots[0]->child(0);
-        for (size_t i = 0; i < m_treeDoc->m_meta_graph->PointMaps::size(); i++) {
-            QString name = QString(m_treeDoc->m_meta_graph->PointMaps::at(i).getName().c_str());
+        for (size_t i = 0; i < m_treeDoc->m_meta_graph->getPointMaps().size(); i++) {

```

Comment:
That iteration looks a bit inefficient. Iterate items in vector<PointMaps> plus counter?